pika/adapters/utils/connection_workflow.py

```diff
@@ -186,7 +187,8 @@ def start(self, addr_record, conn_params, on_done):
         self._sock.setblocking(False)
 
         addr = self._addr_record[4]
-        _LOG.info('Connecting to AMQP broker at %r', addr)
+        _LOG.info('Pika version %s connecting to AMQP broker at %r',
```

Comment:
Can we make this say "node"? "AMQP" is used to refer to two protocols and the target could be a proxy or load balancer, too.